buildAll.bat

```diff
@@ -38,6 +38,8 @@ call build\build.bat vioinput\sys\vioinput.vcxproj "Win10_SDV Win11_SDV" %*
 if errorlevel 1 goto :fail
 call build\build.bat viomem\sys\viomem.vcxproj "Win10_SDV Win11_SDV" %*
 if errorlevel 1 goto :fail
+call build_cab.bat
+if errorlevel 1 goto :fail
```

Comment:
build_cab requires `fwcfg` driver that we don't build at this point. Move it after `for %%D in (pciserial fwcfg Q35) do @(` loop

Reply:
About templates - I prefer not to use it at this time